src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -472,6 +473,7 @@ export const SPECIAL_CURRENCY_INFO: {
     initWalletName: lstrings.string_first_ethereum_pow_wallet_name,
     chainCode: 'ETHW',
     dummyPublicAddress: '0x0d73358506663d484945ba85d0cd435ad610b0a0',
+    fioChainCode: 'ETHEREUMPO',
```

Comment:
Is this really `ETHEREUMPO` not `ETHEREUMPOW`?

Reply:
yes, codes are limited to 10 characters